### DISCUSSION TITLE
Omada DNS not working

### DISCUSSION THREAD

**Discussion by CHROMEOSKILLA** [3kh0/ext-remover#1399] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
yo so I followed all the steps for omada dns using titanium networks guide since ext-removers guide only had one DNS server when chromebooks require 4, and for some reason it wont work. fortiguard intrusion prevention still works even tho the block list omada DNS has clearly includes it as being blocked. is omada DNS even up anymore?

**Comment by Luigi-Master** [comment, marked as answer]
Sadly no, Omada DNS does not work anymore.
